src/bigint.ts

```diff
@@ -0,0 +1,29 @@
+import type { Scalar, ScalarTag } from 'yaml'
+import { StringifyContext, stringifyString } from 'yaml/util'
+
+const identify = (value: any) => {
+  return typeof value === 'bigint' || value instanceof BigInt
+}
```

Comment:
Personally I'd inline this to reduce unnecessary abstraction, but that's just a preference.

Reply:
The indirection saves a bit of extra annotation that typescript requires when using a method in the object initializer (because `identify()` is called in the stringify method). But I don't have a strong preference there.

Reply:
I guess we already had `satisfies ScalarTag` so it doesn't save much anyway, can just `export bigint: ScalarTag = ...`